src/main/scala/mesosphere/marathon/api/v2/InfoResource.scala

```diff
@@ -44,7 +44,7 @@ class InfoResource @Inject() (
     "leader_proxy_read_timeout_ms" -> config.leaderProxyReadTimeout.get,
     "local_port_max" -> config.localPortMax.get,
     "local_port_min" -> config.localPortMin.get,
-    "master" -> config.mesosMaster.get,
+    "master" -> s"zk://${config.mesosMasterZooKeeperHosts}${config.mesosMasterZooKeeperPath}",
```

Comment:
Would it be better to obfuscate it? So that it was clear that credentials were specified?

Reply:
I don't see how it can be helpful. What use-case do you have in mind?

Reply:
Wow, this fell through the cracks. Obfuscating (xxxxxxx) might help communicate that authentication is used.